packages/gatsby/src/schema/resolvers.js

```diff
@@ -93,7 +93,7 @@ const paginate = (results = [], { skip = 0, limit }) => {
   }
 }
 
-const link = ({ by, from }) => async (source, args, context, info) => {
+const link = ({ by = `id`, from }) => async (source, args, context, info) => {
```

Comment:
I wonder if we can/need to generate a unique id (based on timestamp or something) to set as default?

Reply:
@sidharthachatterjee This is about the default node field for foreign-key relations. So that it is possible to say something like this in SDL:
```graphql
type Post implements Node {
  author: AuthorJson @addResolver(type: "link")
  # instead of:
  # author: AuthorJson @addResolver(type: "link", options: { by: "id" })
}
```
The ID itself must already be unique on nodes. Hope that make sense?

Reply:
That makes sense! Thank you for the added context, @stefanprobst